client/state/actions/getMatchTimeline.ts

```diff
@@ -24,7 +24,10 @@ export const getMatchTimeline = ({ username, matchId }: GetMatchTimelineProps):
     dispatch(loadMatchSuccess({ matchData: matchData.data, matchId }));
   } catch (error) {
     dispatch(loadMatchFailure(
-      { message: `Could not fetch match timeline. ${error.response.statusText}: ${error.response.data.error}` || 'An error has occurred' },
+      {
+        message:
+        `Could not fetch match timeline. ${error.response?.statusText}: ${error.response?.data.error}` || 'An error has occurred',
```

Comment:
ideally would not need to check if `response` is undefined, but for the sake of wrapping it up I'll leave as is.